src/components/sections/Hero.section.tsx

```diff
@@ -1,8 +1,95 @@
-const HeroSection = () => {
+import React, { useState, useEffect } from 'react';
+
+
+
+const HeroSection: React.FC = () => {
+  const [countdown, setCountdown] = useState<string>('');
+
+  // Function to calculate the countdown
+  const calculateCountdown = () => {
+    const targetDate = new Date('2024-04-06T00:00:00'); // Set your target date here
+    const currentDate = new Date();
+    const difference = targetDate.getTime() - currentDate.getTime();
+
+    const days = Math.floor(difference / (1000 * 60 * 60 * 24));
+    const hours = Math.floor((difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+    const minutes = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60));
+    const seconds = Math.floor((difference % (1000 * 60)) / 1000);
+
+    setCountdown(`${days} ${hours} ${minutes} ${seconds}`);
+  };
+
+  // Update the countdown every second
+  useEffect(() => {
+    const interval = setInterval(() => {
+      calculateCountdown();
+    }, 1000);
+
+    // Cleanup the interval on component unmount
+    return () => clearInterval(interval);
+  }, []);
+
     return (
-        <>
-        </>
-    );
-};
+        <div className="bg-midnight text-white py-16">
+          <div className="text-center">
+            <h1 className="text-6xl font-bold mb-4">
+             <span className="text-mint">&lt;HawkHacks 2024/&gt;</span>           
+            </h1>
+            <p className="pb-2 text-lg">Wilfrid Laurier’s first large-scale global hackathon!</p>
+            <p className="pb-3 text-xs font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
+            <p className="mx-5 space-x-8 text-xs font-bold [word-spacing:50px]">{countdown}</p>
+            <p className="pb-3 text-sm font-thin [word-spacing:33px]">Days Hrs Mins Secs</p>
+            <button className="bg-mint text-white px-4 py-1 rounded">
+              REGISTER
+            </button>
+            <img src="src/assets/graphic-mountain.png" alt="Mountain Graphic" />
+            <div className="flex items-center justify-center space-x-4 mt-[-8rem]">
+            <a href="#!" role="button">
+              <svg
```

Comment:
Is it possible to move this svg into a separate component? I think it can help with readability when we have a properly named component of this graphic instead of deciphering what is this svg.

Reply:
^